stable/insights-agent/templates/falco/rbac.yaml

```diff
@@ -13,10 +13,31 @@ metadata:
   labels:
     app: insights-agent
 rules:
+  - apiGroups:
+      - 'apps'
+      - 'extensions'
+    resources:
+      - 'deployments'
+      - 'statefulsets'
+      - 'replicasets'
+      - 'daemonsets'
+    verbs:
+      - 'get'
+      - 'list'
+  - apiGroups:
+      - 'batch'
+    resources:
+      - 'jobs'
+      - 'cronjobs'
+    verbs:
+      - 'get'
+      - 'list'
```

Comment:
I think we should make this the `view` clusterrole

Reply:
mmh will `view` able to solve this access issue ? `cannot list resource \"jobs\" in API group \"batch\" in the namespace \"xxx\""`